packages/bpk-component-chip/src/BpkDismissibleChip.js

```diff
@@ -34,18 +35,31 @@ import {
   COMMON_DEFAULT_PROPS,
   type CommonProps,
 } from './commonTypes';
+import STYLES from './BpkSelectableChip.module.scss';
 
 export type Props = CommonProps;
 
+const getClassName = cssModules(STYLES);
+
 const BpkDismissibleChip = (props: Props) => {
-  const { ...rest } = props;
+  const { className, type, ...rest } = props;
+  const iconClassNames = getClassName(
+    `bpk-chip--${type}-dismissible__trailing-accessory-view`,
+  );
+  const dismissibleClassNames = getClassName(
+    `bpk-chip--${type}-dismissible`,
+    className,
+  );
 
   return (
     <BpkSelectableChip
       {...rest}
-      trailingAccessoryView={<CloseCircleIconSm />}
-      selected={false}
+      disabled={false}
```

Comment:
According to the figma design, the Dismissible chip component should not have a disabled state.